### PR TITLE
fix(safari): basic dark mode support

### LINKED ISSUE #201: feat(safari): dark mode color palette
https://pocket.slack.com/archives/CCRGNEGEP/p1569972088297600

### PR DESCRIPTION
**Pull request by dskuza**
Fixes #201

## Goal

Fix #201: basic launcher dark mode support

## Todos:
- [X] Remove explicit background color
- [X] Update assets for any / dark modes

## Implementation Decisions

Assets were taken from design [here](https://pocket.slack.com/archives/CCRGNEGEP/p1570479292328300) and updated within the project to add support for dark mode. By removing the explicit background color, the launcher takes the color of the window based on the selected system theme (i.e white for light, black for dark). 

## All Submissions:

- [ ] Have you followed the guidelines in our Contributing document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### CODE REVIEW

**Comment by dskuza** [comment]
@nzeltzer @collectedmind please add the correct label(s) / project(s) / milestone(s) as required

**Comment by nzeltzer** [comment]
@dskuza Let’s get Jen’s eyes on this before merging.

**Comment by dskuza** [comment]
@nzeltzer Re-review requested as I made a change to explicitly set window color for OSes < 10.14 (when dark mode was introduced).

File: _safari/Save to Pocket/ViewController.swift [hunk 1]
```diff
@@ -72,7 +117,45 @@ class ViewController: NSViewController {
     }
     
     @IBAction func didClickNeedHelpButton(click: NSButton) {
+        needHelp.attributedTitle = needHelpTitle(for: .clicked)
         NSWorkspace.shared.open(URL(string: "https://getpocket.com/howto-pocket-for-safari")!)
     }
+    
+    private func needHelpTitle(for state: NSButton.State) -> NSAttributedString {
```
Comment (nzeltzer) [inline _safari/Save to Pocket/ViewController.swift]: 👍

Comment (nzeltzer) [reply _safari/Save to Pocket/ViewController.swift]: I like how you did this.

**Pull request merged**